### ISSUE TITLE
Add download form functionality

### ISSUE THREAD

**Issue by peterdesmet** [Datafable/epu-index#36]
If the user clicks the download data button (see #5), the a csv file should download with data from the start date to end date (both inclusive)

* [x] If no dates are selected, download all data
* [ ] Data format (tab separated csv):

```
date    epu-index
2014-03-21    3.31
2014-03-22    3.14
```

**Comment by peterdesmet** [comment]
Still unknown:

* Is tab separated preferred (I guess, because it opens directly in Excel)
* How should epu-index be rounded?
* Is `.` or `,`?

**Comment by bartaelterman** [comment]
A lot of the work is already done. See #12 .

**Comment by peterdesmet** [comment]
Blocked by #45

**Comment by peterdesmet** [comment]
* The desired data is now returned by the API (#45), including number of newspapers and number of articles.
* The data are comma separated, which makes it a bit tricker to copy and paste to Excel, but I think it is is more standard (ask user maybe?)
* The download form is implemented in a more simple way (as links) in #5.

I'll close this issue.
